stix_shifter_modules/azure_sentinel/stix_translation/json/from_stix_map.json

```diff
@@ -1,7 +1,15 @@
 {
   "ipv4-addr": {
     "fields": {
-      "value": ["networkConnections.sourceAddress", "networkConnections.destinationAddress", "hostStates.publicIpAddress", "hostStates.privateIpAddress", "userStates.logonIp"]
+      "value": [
+        "networkConnections.sourceAddress",
+        "networkConnections.destinationAddress",
+        "hostStates.publicIpAddress",
+        "hostStates.privateIpAddress",
+        "userStates.logonIp",
```

Comment:
remove "hostStates.publicIpAddress", "hostStates.privateIpAddress", "userStates.logonIp" from `"ipv4-addr"` and add those as `x-msazure-sentinel-alert` custom objects

Reply:
removed